tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -30,6 +30,13 @@ class OriginOptions(Enum):
     DOCUMENT = "document"
     VIEWPORT = "viewport"
 
+class FormatOptions(Dict[str, Any]):
+
+    def __init__(self, type: str, quality: Optional[float] = None):
+        self["type"] = type
```

Comment:
We should correctly initialize the class as dict, similar to the other custom classes in other modules:

```suggestion
    def __init__(self, type: str, quality: Optional[float] = None):
        dict.__init__(self, type=type)

```